charts/etcd-copy-backups/templates/etcd-copy-backups-job.yaml

```diff
@@ -0,0 +1,246 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    gardener.cloud/owned-by: "{{ .Release.Namespace }}/{{ .Values.ownerName }}"
+    gardener.cloud/owner-type: "etcdcopybackupstask"
+{{- if .Values.annotations }}
+{{ toYaml .Values.annotations | indent 4 }}
+{{- end }}
+  labels:
+    networking.gardener.cloud/to-public-networks: allowed
```

Comment:
Do we need this label?

Reply:
We do, since the actual copying of backups by `etcd-backup-restore` needs to access the backup containers on the cloud providers. Or am I missing something?

Reply:
To my understanding this is set via `spec.template.metadata.labels`, right?

Reply:
Right :-( Not sure what this label was doing here, it looks like another leftover from before that I didn't clean. Thanks for noticing!